test/controllers/ProcessReviewControllerSpec.scala

```diff
@@ -867,27 +795,71 @@ class ProcessReviewControllerSpec extends WordSpec with Matchers with ScalaFutur
 
     "the request is valid" should {
 
-      trait ValidTest extends Test {
-
+      "return a NO_CONTENT response" in new Test {
         MockReviewService
           .approvalPageComplete("id", "/pageUrl", ReviewTypeFactCheck, reviewUpdate)
           .returns(Future.successful(Right(())))
 
         lazy val request: FakeRequest[JsValue] = FakeRequest().withBody(Json.toJson(reviewUpdate))
-      }
-
-      "return a NO_CONTENT response" in new ValidTest {
         private val result = controller.approvalFactCheckPageComplete(reviewUpdate.id, "pageUrl")(request)
         status(result) shouldBe NO_CONTENT
-      }
-
-      "return no content" in new ValidTest {
-        private val result = controller.approvalFactCheckPageComplete(reviewUpdate.id, "pageUrl")(request)
         contentType(result) shouldBe None
       }
 
     }
 
   }
 
+  def createProcess: JsObject = {
```

Comment:
I would be tempted to put this hardcoded process into a separate file. Similar to PrototypeJson in view-external-guidance-frontend. Further, if you really want a valid process I would add a PageStanza with the id "start". Probably - "start": {
      "next": [
        "1"
      ],
      "stack": false,
      "type": "PageStanza",
      "url": "/example-page-1"
    },

Reply:
This just needs to be a process that meets the format of a valid ocelot process, the test doesn't care about the content, just format.  Spoke to Chris a while ago about test data and thought better to put data in test where needed as makes tests less fragile if someone needs to change the "central" test data.

Reply:
Ok, I guess it is a fairly subjective issue.